crowbar_framework/app/controllers/installers_controller.rb

```diff
@@ -42,27 +46,74 @@ def status
   # Provides the restful api call for
   # /installer/start 	POST 	triggers installation
   def start
```

Comment:
A lot of code duplication, maybe you should create some variables and do the respond_to block only at the bottom of the method?